coordinape/coordinape-backend
# Add env vars to toggle specific crons

### Test Plan 

1. add the env vars
```
CRON_NOMINEE_ENABLED=false
CRON_DAILY_UPDATE=false
CRON_EPOCH_NOTIF=false
CRON_EPOCH_CLOSE=true
```
2. trigger the cron job manually by running using its signature e.g `php artisan check:nominees`
3. check if env var is correctly blocking the cron job
